src/urllib3/response.py

```diff
@@ -443,9 +444,8 @@ def _error_catcher(self):
             except BaseSSLError as e:
                 # FIXME: Is there a better way to differentiate between SSLErrors?
                 if "read operation timed out" not in str(e):  # Defensive:
```

Comment:
Can we remove that Defensive pragma? Without that, I think we won't get coverage information for that condition.

Reply:
Oh yeah, I'll remove that :) Nice catch!